### PR TITLE
IDE-3401 add scripts for removing and checking for linux and Mac inst…

### PR DESCRIPTION
**Pull request by yanan**
…allers

### CODE REVIEW

**Comment by yanan** [comment]
Including Ying‘s windows part already, please help check and merge.

**Comment by jtydhr88** [comment]
@yanan @oyangbingxi 
please do the following changes:
1. use the name **studio-installer-checker-linux.sh** or not **studio-installers-checker-linux.sh**, (remove the 's') same as another files.

File: build/installers/scripts/linux/studio-installers-checker-linux.sh [hunk 1]
```diff
@@ -0,0 +1,106 @@
+#Add all properties
+homeDir="/home/liferay"
```
Comment (jtydhr88) [inline build/installers/scripts/linux/studio-installers-checker-linux.sh]: should not hardcode this home dir
please try to use **$HOME** directly

File: build/installers/scripts/linux/studio-installers-checker-linux.sh [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+exceptedBladeVersion="2.2.0.201707061805"
+exceptedJpmVersion="3.2.0.201605172009"
+bundleName="liferay-dxp-digital-enterprise-tomcat-7.0-sp4-20170705142422877.zip"
+liferayWsDir="/home/liferay/LiferayDeveloperStudio/liferay-workspace"
+studioDir="/home/liferay/LiferayDeveloperStudio"
```
Comment (jtydhr88) [inline build/installers/scripts/linux/studio-installers-checker-linux.sh]: same as **homeDir**

File: build/installers/scripts/linux/studio-installers-checker-linux.sh [hunk 3]
```diff
@@ -0,0 +1,106 @@
+#Add all properties
+homeDir="/home/liferay"
+exceptedBladeVersion="2.2.0.201707061805"
+exceptedJpmVersion="3.2.0.201605172009"
+bundleName="liferay-dxp-digital-enterprise-tomcat-7.0-sp4-20170705142422877.zip"
+liferayWsDir="/home/liferay/LiferayDeveloperStudio/liferay-workspace"
```
Comment (jtydhr88) [inline build/installers/scripts/linux/studio-installers-checker-linux.sh]: same as **homeDIr**

File: build/installers/scripts/linux/studio-installers-remover-linux.sh [hunk 4]
```diff
@@ -0,0 +1,21 @@
+#Add properties here
+workspaceDir="/home/liferay/LiferayDeveloperStudio/liferay-workspace"
```
Comment (jtydhr88) [inline build/installers/scripts/linux/studio-installers-remover-linux.sh]: no hardcode here

**Comment by jtydhr88** [comment]
2. please not hardcode the **homeDir**

**Comment by jtydhr88** [comment]
3. let us use the the following structure for liunx and mac:
srciprts\
  \common\env.sh   -------- set the common variables that can be share like **homeDir**, **version** and etc.
  \liunx\env-linux.sh -------  set the linux special variables that like **jpmHome** and etc 
  \mac\env-mac.sh -------  set the mac special variables that like **jpmHome** and etc 

btw you can use **source** to include one shell in another shell script

File: build/installers/scripts/windows/studio-installers-checker-windows.bat [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+set user.home.dir=C:\Users\liferay
+set liferay.workspace.home.dir="C:\Program Files (x86)\LiferayDeveloperStudio\liferay-workspace"
+set liferay.developer.studio.home.dir="C:\Program Files (x86)\LiferayDeveloperStudio\liferay-developer-studio"
+set start.menu.program.dir="C:\ProgramData\Microsoft\Windows\Start Menu\Programs"
+set sdk.bundle.zip=com.liferay.portal.plugins.sdk-1.0.11-withdependencies.zip
+set portal.bundle.zip=liferay-dxp-digital-enterprise-tomcat-7.0-sp4-20170705142422877.zip
+set expect.blade.version=2.2.0.201707061805
+
+::check ${system_temp_directory} does not exist
+if not exist %user.home.dir%\AppData\Local\Temp\LiferayWorkspace (echo The LiferayWorkspace folder doesn't exist.^(Successfully^)) else echo The LiferayWorkspace folder exists.(Error)
```
Comment (jtydhr88) [inline build/installers/scripts/windows/studio-installers-checker-windows.bat]: please make sure this command works well for both win7 and win10

Comment (jtydhr88) [inline build/installers/scripts/windows/studio-installers-checker-windows.bat]: And please use system command to get temp folder or not hardcode here

File: build/installers/scripts/windows/studio-installers-checker-windows.bat [hunk 6]
```diff
@@ -0,0 +1,51 @@
+@echo off
+::add all variable
+set user.home.dir=C:\Users\liferay
```
Comment (jtydhr88) [inline build/installers/scripts/windows/studio-installers-checker-windows.bat]: no hardcode here

**Comment by jtydhr88** [comment]
see comments. Thanks!

**Pull request closed without merging**